cmd/prometheus-config-reloader/main.go

```diff
@@ -91,6 +97,12 @@ func main() {
 
 	versionutil.RegisterIntoKingpinFlags(app)
 
+	err := web.Validate(*webConfig)
```

Comment:
hmm this should happen after? ideally after the logging has been setup.

Reply:
Thanks for your review. 
Done. Move after the logging setuped